python_ta/contracts/__init__.py

```diff
@@ -176,6 +180,15 @@ def _check_function_contracts(wrapped, instance, args, kwargs):
     return r
 
 
+def _get_argument_suggestions(arg: Any, annotation: type) -> str:
+    """Returns potential suggestions for the given arg and its annotation"""
+    suggestions = ""
+    if isinstance(arg, type) and annotation in arg.mro():
+        suggestions += f'Did you mean to pass in an instance of {arg}?'
```

Comment:
I think we have this problem somewhere else, but it's better to use `arg.__name__`, which evaluates to just the name of the type.

I'm wondering if this message should be a little more explicit about how the code should change. It might be good to think about a concrete example and run the text by the other PyTA team members.

> Did you mean to pass in an instance of Player?

> Did you pass in Player instead of Player(...)?

> Did you forget the parentheses after Player?

etc. Do some brainstorming here!